crates/evm/src/evm/handler.rs

```diff
@@ -619,6 +619,7 @@ fn decrease_caller_balance<EXT, DB: Database>(
     change_balance(context, amount, false, address)
 }
 
+#[allow(unused)]
```

Comment:
this should be used?

Reply:
and yes, it is used.

in query.rs line 834

Reply:
it is not used in zk code, gives warning in build. was in between this and putting it behind native feature, chose unused.

Reply:
native makes more sense actually. do it pls

Reply:
done